src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -436,15 +436,17 @@
             },
             "preview": {
                 "label": "Preview",
-                "tooltip": "For every programming exercise, repositories and build plans will be generated. Here you can find their generated names based on the short name of the exercise (and course)",
+                "tooltip": "For every programming exercise, repositories and build plans will be generated. Here, you can find their generated names based on the short name of the exercise (and course)",
+                "tooltipLocalVC": "Here, you can find the repositories linked to the exercise and their respective checkout directories during the Build Execution.",
                 "repositories": "Repositories",
                 "buildPlans": "Build Plans",
                 "templateRepoTooltip": "The repository which contains the template of the exercise",
                 "solutionRepoTooltip": "The repository which contains the solution to the exercise",
                 "testRepoTooltip": "The repository which contains all tests",
                 "auxiliaryRepoTooltip": "{{description}}",
                 "templateBuildPlanTooltip": "The build plan of the template. Runs all tests on the exercise template. Used as a reference in which all tests must fail",
-                "solutionBuildPlanTooltip": "The build plan of the solution. Runs all tests on the exercise solution. Used as a reference in which all tests must succeed"
+                "solutionBuildPlanTooltip": "The build plan of the solution. Runs all tests on the exercise solution. Used as a reference in which all tests must succeed",
+                "checkoutDirectoryDuringBuildExecution": "Checkout Directory during Build Execution"
```

Comment:
Should this be in title case?

Reply:
It as intentional but I do not have a strong opinion on the capitalization